README.md

```diff
@@ -0,0 +1,20 @@
+##  README.md
+
+### Purpose of this repository
+
+We want to provide you some very simple examples of how to use ReSim. This repository is provided as a simple, stripped down example so you can walk through how to get ReSim up and running before specifically getting your code ready for ReSim. 
+
+### Areas to be highlighted
+
+This Getting Started guide will highlight the following areas:
+
+1. Providing an experience builder that allows you to create a simple scene
+2. Creating a metrics that build that takes the outputs of the experience builder and creates a simple metrics
```

Comment:
```suggestion
1. Providing an experience build that allows you to create a simple scene
2. Creating a metrics build that takes the outputs of the experience build and creates a simple metric
```

I'm also not sure that the existing experience build can be said to be creating a scene.